db/data/20200814135953_report_states_are_strings.rb

```diff
@@ -0,0 +1,9 @@
+class ReportStatesAreStrings < ActiveRecord::Migration[6.0]
+  def up
+    Report.update_all state: "inactive"
```

Comment:
I'm always a bit wary of having a one-way destructive migrations, but probably ok given how this field isn't really populated right now.

Reply:
Yeah - this one worries me, I think this is the only time we can do it and my understanding of the data migrations is that this will only evert run once.

Reply:
I only just spotted that this was a data migration rather than a regular one! :-(